### PR TITLE
Add content on remotes, documentation and the workshop project

### LINKED ISSUE #1: Create first version of slides
For further discussion.

### PR DESCRIPTION
**Pull request by fsteeg**
Deployed to http://hbz.github.io/swib16-workshop

Will partially resolve #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,23 +212,47 @@
...
+						<h2>Exercise 4-b:<br/>README formatting</h2>
+						<p><span class="fragment">Rename the README file to README.md</span></p>
+						<p><span class="fragment">Add headers, styling, and details to your README content</span></p>
+						<p><span class="fragment"><a href="https://help.github.com/articles/basic-writing-and-formatting-syntax/">http://help.github.com/articles/basic-writing-and-formatting-syntax</a></span></p>
 					</section>
 					<section>
 						<h1>GitHub pages</h1>
-						<p>TODO</p>
+						<p><span class="fragment">GitHub also provides hosting of static HTML for your repo</span></p>
+						<p><span class="fragment">Pages are available at <br/><code>http://&lt;username&gt;.github.io/&lt;repo-name&gt;</code></span></p>
```
Comment (acka47) [inline index.html:239]: Correctly, we should also mention organisations here: `<code>http://&lt;user/organisation&gt;.github.io/&lt;repo-name&gt;</code>`

 I am not sure whether it adds unnecessary complexity, though.

**Comment by acka47** [comment]
I committed some small improvements. Generally, I really like the structure and how exercise and lessons built on each other. 

We should think about facilitating the understanding of different types slides by better distinguishing them visually, see #14. 

Another small thing I noticed: regarding the process in https://hbz.github.io/swib16-workshop/#/15/4 we should make clear that this is _one_ possible way of many for structuring a Kanban and that it is the approach we chose.

**Comment by acka47** [comment]
Please merge if you are ok with my adjustments.

File: index.html (hunk 1 above)
Comment (fsteeg) [reply index.html:239]: And then there are also the org/user pages themselves (not for repos). I think it's better to keep it simple here. We also link to the complete documentation, so people can find out about these things. And it's probably something we'll mention anyway.

**Comment by fsteeg** [comment]
I find the 'non-developer' term a bit odd, I think because it's kind of dev-centric by defining all other roles as non-dev, but I see your point. Will merge for now, maybe we'll find something better later.

**Pull request merged**